src/components/common/SocialMediaContainer/SocialMediaContainer.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import 'components/common/SocialMediaContainer/social-media-container.scss';
```

Comment:
I think you should import the scss from the index file of the styles, not here

Reply:
Cool, seems better to me. I moved every scss file import to the main styles.scss file. I think there's a way to import every .scss inside the components/ and pages/ folder, something like @import '../components/**/*.scss', but somehow I didn't get it to work.